### DISCUSSION TITLE
JSON Schema Generation

### DISCUSSION THREAD

**Discussion by ainsleyclark** [payloadcms/payload#7083] [Feature Requests & Ideas]
It would be **incredibly** useful if there was a command to generate JSON schema so we're able to generate types to interact with that's not necessarily TS.

`payload generate:types`

I'm aware that Payload currently converts to JSON then TS types so I think this will be quite minimal.

At the moment I'm forking the `configToJSONSchema` function and it's quite cumbersome.

**Comment by denolfe** [comment]
This should now be possible in v3 after this PR was merged: https://github.com/payloadcms/payload/pull/6984

You can define a function and perform whatever modifications you'd like.

**Comment by ainsleyclark** [comment]
Hi @denolfe thanks for your reply.

I understand that this modifies the Payload JSON schema but from my understanding there is still no way to generate a `json` file (`JSON Schema`)?
